src/main/scala/view/scalaFX/ScalaFXView.scala

```diff
@@ -58,3 +64,75 @@ object ScalaFXView extends View {
 
   override def handleBunnyClick(bunny: BunnyView): Unit = baseAppController --> { _.handleBunnyClick(bunny) }
 }
+
+object ScalaFxViewConstants {
```

Comment:
Questo oggetto l'abbiamo messo in un file separato, perciò va tolto da ScalaFXView